### PR TITLE
[hotfix] Fixes broken TopSpeedWindowing scala example

### PR DESCRIPTION
**Pull request by kl0u**
The problem was that in the case where no input file was provided, the `fromCollection()` source was never exiting and the actual program was never running. The new source is identical to the corresponding java test.

### CODE REVIEW

File: flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples/windowing/TopSpeedWindowing.scala [hunk 1]
```diff
@@ -72,7 +74,34 @@ object TopSpeedWindowing {
...
+
+          val speeds = Array.fill[Integer](numOfCars)(50)
+          val distances = Array.fill[Double](numOfCars)(0d)
+          @Transient lazy val rand = new Random()
+
+          var counter = 0
+          var isRunning:Boolean = true
+
+          override def run(ctx: SourceContext[CarEvent]) = {
+            while (isRunning && counter < numOfEvents) {
```
Comment (tillrohrmann) [inline flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples/windowing/TopSpeedWindowing.scala]: I think we shouldn't limit the number of emitted events. It is a streaming example and as such it should continuously run.

Comment (kl0u) [inline flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples/windowing/TopSpeedWindowing.scala]: The reason for this is to be the same as the java TopSpeedWindowing example. 
I suggest that if we change the one, we should also change the other too, as the user would expect the same behavior from both.

What do you think?

**Comment by tillrohrmann** [comment]
Good changes @kl0u. I only had a minor comment.

File: flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples/windowing/TopSpeedWindowing.scala (hunk 1 above)
Comment (tillrohrmann) [inline flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples/windowing/TopSpeedWindowing.scala]: Yes you're right. Then let's also change the java example.

**Comment by kl0u** [comment]
@tillrohrmann  I integrated the comments. 
Please review.

**Comment by tillrohrmann** [comment]
Changes look good to me. Thanks for your work @kl0u. Will merge this PR.

**Pull request closed without merging**